packages/integration-tests/api/api.js

```diff
@@ -58,18 +59,12 @@ async function getApiOauthProvider(prefix) {
  *   name.
  * @param {string} params.payload - the payload to send to the Lambda function.
  *   See https://docs.aws.amazon.com/apigateway/latest/developerguide/set-up-lambda-proxy-integrations.html#api-gateway-simple-proxy-for-lambda-input-format
- * @param {Object} params.userParams - parameter object passed to fakeUserFactory
  * @returns {Promise<Object>} - the parsed payload of the response.  See
  *   https://docs.aws.amazon.com/apigateway/latest/developerguide/set-up-lambda-proxy-integrations.html#api-gateway-simple-proxy-for-lambda-output-format
  */
-async function callCumulusApi({ prefix, payload: userPayload, userParams = {} }) {
+async function callCumulusApi({ prefix, payload: userPayload }) {
```

Comment:
I can't recall where, but I believe `userParams` was being used to override the user authenticated for this request. Perhaps in some of the distribution tests?

Reply:
I can't find any code that calls `callCumulusApi` and passes userParams.

Reply:
Nevermind, I was thinking of a different function it seems